cfgov/unprocessed/css/pages/beta.less

```diff
@@ -13,10 +13,14 @@
     font-size: 0.875em;
     transition: padding 0.2s ease-out;
 
-    &.expandable__expanded {
+    &.m-expandable__expanded {
         padding: @grid_gutter-width 0;
```

Comment:
This is fine for now, but I had a discussion with @jimmynotjim that we'll likely want to abstract out some kind of generic expandable stuff so the atomic expandable isn't being customized outside of the atomic workflow (customizing margin is fine, but changing other parts of the appearance arguably changes it into an atomic element inconsistent with the design docs). We'd talked about it in terms of JS to have something like:

```
           GenericExpandableThing.js
             /         \           \
Expandable.js    SecondaryNav.js    BetaBanner.js
```
_Note: above is using composition **not** inheritance_

The same sort of structure may make sense for the CSS as well. Really food for thought at this point though.